convert_osm/src/lib.rs

```diff
@@ -111,9 +111,9 @@ pub fn convert(opts: Options, timer: &mut abstutil::Timer) -> RawMap {
     parking::apply_parking(&mut map, &opts, timer);
 
     // TODO Make this bail out on failure, after the new dependencies are clearly explained.
-    if let Err(err) = elevation::add_data(&mut map, timer) {
+    /*if let Err(err) = elevation::add_data(&mut map, timer) {
```

Comment:
Will revert this; I introduced an unrelated bug earlier in elevation and need to fix it up.